eslint.config.mjs

```
@@ -0,0 +1,39 @@
+  ),
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser
+      },
+
+      ecmaVersion: 5,
+      sourceType: 'module',
+
+      parserOptions: {
+        parser: '@typescript-eslint/parser'
```

ESLint

[error] Replace `'@typescript-eslint/parser'` with `"@typescript-eslint/parser",`

(prettier/prettier)